src/Apps/Order/Components/AddressForm.tsx

```diff
@@ -209,9 +209,10 @@ export class AddressForm extends React.Component<
               <Input
                 id="AddressForm_phoneNumber"
                 title="Phone number"
+                type="tel"
                 description={this.phoneNumberInputDescription()}
-                placeholder="Add phone"
-                pattern="[0-9]*"
+                placeholder="+1-222-222-2222"
```

Comment:
Is this desired from a design perspective? It's quite different from our other placeholders.

Reply:
I just choose this a bit randomly. I didn't see anything in palette here, but happy to replace with anything more consistent. Could you point me in a direction if you have an idea in mind here?

Reply:
I believe `Add phone` (what was there) was intentional (and matches the other placeholders).

I recommend asking the design team if you'd like to change!

Reply:
Gotcha – I guess I presumed that this wasn't a deliberate decision, but good to know that it was. I'll revert the placeholder change as that isn't the thrust of this PR and don't want to slow down the fix.

Reply:
FYI, I brought this up at the Design Systems meeting today and we're going to revisit the placeholder copy which I'm hoping will either clarify the intent or open a discussion towards alternatives: https://artsyproduct.atlassian.net/browse/DSG-15 :lock:

Reply:
Very cool, thanks for doing that!